runner/runner/main.py

```diff
@@ -87,27 +91,28 @@ async def subprocess_run(
     return proc
 
 
-async def make(path: Path, target: str, var_dict: Optional[Dict[str, str]] = None) -> None:
+async def make(path: Path, targets: List[str], var_dict: Optional[Dict[str, str]] = None) -> None:
```

Comment:
`make` now takes a _list_ of targets, since GNU Make supports building multiple targets simultaneously more efficiently than one-after-another.